_includes/configuration/formats.md

```diff
@@ -327,7 +328,7 @@ tinymce.init({
 
 Enables control for removing the child elements of the matching format. This is set to `false`  by default on selector formats. As a result, when a class is removed from a selected table class, disabling `deep` retains the class in the child elements within the other nested tables.
```

Comment:
I'm not sure this section about "deep", especially the example, is correct. @spocke Could you or someone else in your team please check this?